topics/motorola-68000/index.md

```diff
@@ -0,0 +1,17 @@
+---
+aliases: m68k, 68k, m68000, 68000, 68000-series, 68000-family, 68000-processor, mc68000, motorola-680x0
+created_by: Motorola
+display_name: Motorola 68000
+related: 68k-assembly, assembly-68000, m68k-atari-mint, m68k-assembly, m68k-disassembler, m68k-assembly, m68k-elf-gcc, assembly-m68k, m68k-emulation, motorola-6805
+
```

Comment:
`m68k-assembly` is still duplicated and there is still a blank line, removing both here:
```suggestion
related: 68k-assembly, assembly-68000, m68k-atari-mint, m68k-assembly, m68k-disassembler, m68k-elf-gcc, assembly-m68k, m68k-emulation, motorola-6805
```